### PR TITLE
#99 Fix PHP Stan error with DrupalAutoloader.

### PR DESCRIPTION
**Pull request by hkirsman**
## Overview

This is trivial change atm as we've just disabled Psalm in previous PR https://github.com/wunderio/code-quality/pull/97 We hope to get Psalm working again at some point.

There's failure in main branch tests https://app.circleci.com/pipelines/github/wunderio/code-quality/257/workflows/ba91db7e-cc48-45a4-ac72-ce8cc7530c7d/jobs/300?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary
 
This PR fixes it. After this we can create 3.0 for Code Quality and have GrumPHP 2.5 in our projects!

## Screenshots

image

image

## Testing

1. Require this branch in your project:
`composer require wunderio/code-quality:dev-hotfix/99-fix-php_stan-error-with-DrupalAutoloader --dev --with-all-dependencies`

2. Disable psalm in grumphp.yml or otherwise you'll get "The executable for "psalm" could not be found."

image

3. Run Code Quality. It might be that you get error about psalm and that's because you still have psalm enabled
```
lando grumphp run
```
or
```
ddev grumphp run
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -44,7 +44,8 @@
         "dealerdirect/phpcodesniffer-composer-installer": "*",
         "mglaman/phpstan-drupal": "^1.1",
         "phpstan/extension-installer": "^1.1",
-        "phpstan/phpstan-deprecation-rules": "^1.0"
+        "phpstan/phpstan-deprecation-rules": "^1.0",
+        "webflo/drupal-finder": "^1.3"
```
Comment (hkirsman) [inline composer.json:48]: I'm also adding requirement to version as the new class was added in 1.3 https://github.com/webflo/drupal-finder/compare/1.2.2...1.3.0#diff-19dafcffb2094365c57f581c138aede2762e021675c6d8dab34c5dbb78508128R1

File: src/Task/Psalm/services.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+services:
```
Comment (hkirsman) [inline src/Task/Psalm/services.yaml:1]: This is not related to current PR. It's something I missed in the GrumpPHP 2.5 update - just a nasty error if psalm task is enabled in grumphp.yml:
image

**Review by tormi — APPROVED** [review]
Thanks, @hkirsman !
